.github/workflows/release_pod.yml

```diff
@@ -0,0 +1,66 @@
+name: Build and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      versionBumpLevel:
+        description: 'Version bump level (patch, minor, major)'
+        required: true
+        type: choice
+        default: 'patch'
+        options:
+        - patch
+        - minor
+        - major
+
+jobs:
+  build-and-release:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      
+      - name: Set up XCode 
+        run: sudo xcode-select --switch /Applications/Xcode_15.0.app
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
+
+      - name: Set up Cocoapods
+        run: gem install cocoapods
+
+      - name: Bump version
+        run: ruby ./scripts/bump_pod_version.rb ${{ github.event.inputs.versionBumpLevel }}
+
+      - name: Build XCFramework
+        run: ./scripts/build_framework.sh
+
+      - name: Get new version
+        id: version
+        run: echo "VERSION=$(ruby -e 'puts File.read("./OSBarcodeLib.podspec").match(/spec.version.*=.*\"(\d+\.\d+\.\d+)\"/)[1]')" >> $GITHUB_ENV
+
+      - name: Create Release
+        id: create_release
+        uses: softprops/action-gh-release@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: v${{ env.VERSION }}
+          name: v${{ env.VERSION }}
```

Comment:
As you can see from our existing Release and Tag, we use the `major.minor.patch` naming convention (without the `v`).